openquake/qa_tests_data/event_based/case_7/expected/hazard_curve-mean.csv

```diff
@@ -1,4 +1,4 @@
-0.00000 1.00000,8.1437737E-01 7.8745190E-01 7.5781661E-01 7.2858428E-01 6.8730971E-01 6.1812530E-01 5.1277028E-01 3.7436129E-01 2.3809462E-01 1.4035469E-01 7.5580318E-02 3.9611371E-02 1.8548869E-02 8.7526477E-03 3.0707836E-03 1.0390706E-03 0.0000000E+00 0.0000000E+00 0.0000000E+00,8.4730505E-01 8.2930087E-01 8.0627473E-01 7.8004131E-01 7.5338795E-01 7.2254874E-01 6.8571804E-01 6.2404352E-01 5.2629674E-01 4.0545358E-01 2.7064192E-01 1.6023936E-01 9.5137381E-02 5.3809276E-02 2.7425979E-02 1.2220817E-02 4.5129637E-03 1.0887562E-03 7.0956520E-04
-0.00000 2.50000,6.0848054E-01 5.1563831E-01 3.8789253E-01 2.4164486E-01 1.1804985E-01 4.6072748E-02 1.2248499E-02 1.0688061E-03 6.2968510E-04 6.2968510E-04 5.4972509E-04 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00,6.8508968E-01 6.4853067E-01 5.8948038E-01 4.9228387E-01 3.5677048E-01 2.1359770E-01 1.0557315E-01 3.4870127E-02 1.0288616E-02 3.3403445E-03 1.0989007E-03 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00
+0.00000 0.00000,8.5935237E-01 8.2641584E-01 7.8489965E-01 7.4466379E-01 6.9942168E-01 6.2868008E-01 5.1680451E-01 3.6587938E-01 2.2947963E-01 1.4543851E-01 7.9506282E-02 4.4411256E-02 2.0331338E-02 8.4450365E-03 4.5833453E-03 1.8566144E-03 8.1918055E-04 3.2983505E-04 0.0000000E+00,8.9884449E-01 8.7611594E-01 8.4806186E-01 8.1424968E-01 7.7648177E-01 7.3610586E-01 6.9216543E-01 6.2567447E-01 5.2197953E-01 4.0022609E-01 2.6916016E-01 1.6204166E-01 9.4660063E-02 5.7920085E-02 3.1158115E-02 1.5938010E-02 7.2895019E-03 2.9013342E-03 4.0979507E-04
+0.00000 1.00000,8.1712654E-01 7.8995929E-01 7.5859713E-01 7.2065396E-01 6.7792359E-01 6.1310979E-01 5.0606272E-01 3.6841306E-01 2.4286609E-01 1.5123441E-01 7.8646755E-02 3.4762479E-02 1.6618300E-02 7.2526026E-03 3.0707836E-03 1.3878477E-03 0.0000000E+00 0.0000000E+00 0.0000000E+00,8.5031363E-01 8.3037928E-01 8.0714866E-01 7.8087136E-01 7.4963580E-01 7.1754637E-01 6.7747697E-01 6.1787004E-01 5.2001434E-01 3.9718247E-01 2.7310472E-01 1.7546872E-01 1.0138969E-01 6.2703556E-02 3.0531054E-02 1.3713041E-02 5.4034178E-03 1.7969143E-03 7.0956520E-04
+0.00000 2.50000,6.0986540E-01 5.1536851E-01 3.8562112E-01 2.3938177E-01 1.1598324E-01 4.8509312E-02 1.5884493E-02 4.2072912E-03 1.6475275E-03 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00,6.8524997E-01 6.4816312E-01 5.8752199E-01 4.8521846E-01 3.5791290E-01 2.2090211E-01 1.0904116E-01 4.3031374E-02 1.1852730E-02 2.0553484E-03 6.2968510E-04 6.2968510E-04 5.4972509E-04 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00
```

Comment:
It looks like the first columns of the two files are more similar to each other, with respect to the final columns. Is this just a coincidence, or is it somehow expected?